cerberus_anymal_b_control_1/src/cerberus_anymal_b_control_1.cpp

```diff
@@ -0,0 +1,34 @@
+/*!
+ * @name      cerberus_anymal_b_control_1.cpp
+ * @author    Samuel Zimmermann
+ * @email     zsamuel@ethz.ch
+ * @date      May 1, 2020
+ *
+ * Copyright (C) 2020 Robotic Systems Lab, ETH Zurich.
+ * All rights reserved.
+ * http://www.rsl.ethz.ch/
+ */
+
+// cerberus_anymal_control_1
+#include "cerberus_anymal_b_control_1/CerberusAnymalBControl1.hpp"
+
+int main(int argc, char** argv) {
+  // Initialize ROS.
+  ros::init(argc, argv, "cerberus_anymal_b_control_ros");
+  ros::NodeHandle nodeHandle;
+
+  // Create the controller
+  cerberus_anymal_b_control_1::CerberusAnymalBControl1 controller(nodeHandle);
+
+  // Set the controller frequency
+  ros::Rate loop_rate(400);
```

Comment:
consider to have a rosparam for this

Reply:
I agree in general. However we don't really want to change this as everything is based on this 400 Hz.